packages/camera/camera_android_camerax/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## 0.6.5+6
```

Comment:
```suggestion
## 0.6.5+7
```

Reply:
Oops, adjusted some minor version changes back down to bugfix manually, and messed this one up!